### PR TITLE
[CLOUDCRAFT-389] Activate your AWS Marketplace Cloudcraft subscription

### PR DESCRIPTION
**Pull request by jamesponddotco**
### What does this PR do? What is the motivation?
Add a new article to the Cloudcraft documentation describing how a customer can activate their Cloudcraft subscription bought through the AWS Marketplace.

### Merge instructions
Please wait until the Cloudcraft team has reviewed the content before going ahead with the review and merging of the article.

- [x] Please merge after reviewing

### CODE REVIEW

**Review by srividya-rajagopalan — APPROVED** [review]

File: content/en/cloudcraft/getting-started/activate-aws-marketplace-cloudcraft-subscription.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Activate your AWS Marketplace Cloudcraft subscription
+kind: documentation
+---
+
+## Overview
+
+[A Cloudcraft subscription can be bought through the AWS Marketplace][1]. This guide will show you how to activate your Cloudcraft subscription after you have purchased it.
```
Comment (maycmlee) [inline content/en/cloudcraft/getting-started/activate-aws-marketplace-cloudcraft-subscription.md:8]:
```suggestion
A Cloudcraft subscription can be bought through the [AWS Marketplace][1]. This guide shows you how to activate your Cloudcraft subscription after you have purchased it.
```

File: content/en/cloudcraft/getting-started/activate-aws-marketplace-cloudcraft-subscription.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+## Activate your subscription
+
+1. In the **AWS Console**, navigate to [the **AWS Marketplace** page][2].
+2. Click the **Manage subscriptions** button in the **Get started** section.
```
Comment (maycmlee) [inline content/en/cloudcraft/getting-started/activate-aws-marketplace-cloudcraft-subscription.md:13]:
```suggestion
2. Click **Manage subscriptions** in the **Get started** section.
```

**Review by maycmlee — COMMENTED** [review]
Left some feedback!

**Comment by jamesponddotco** [comment]
@maycmlee All set! :)

**Review by maycmlee — COMMENTED** [review]
Thanks! Will merge once all the checks have passed.

**Review by maycmlee — APPROVED** [review]

**Pull request merged**